01-Authorization-RS256/composer.json

```diff
@@ -36,14 +36,6 @@
         ],
         "post-create-project-cmd": [
             "php artisan key:generate"
-        ],
-        "post-install-cmd": [
-            "Illuminate\\Foundation\\ComposerScripts::postInstall",
-            "php artisan optimize"
-        ],
-        "post-update-cmd": [
-            "Illuminate\\Foundation\\ComposerScripts::postUpdate",
-            "php artisan optimize"
```

Comment:
These post-install and -update commands run route caching, which will not work with closure-based routes. Since this is a sample, best to keep the example simple by using a closure for the API route and not run any unnecessary optimizations.